CONTRIBUTING.txt

```diff
@@ -222,13 +221,13 @@ Guidelines
 * All code should have tests (see `test coverage`_ below for more details).
 * All code should be documented, to the same
   `standard <https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>`_ as NumPy and SciPy.
-* For new functionality, always add an example to the gallery.
+* For new functionality, always add an example to the gallery (see
+  sphinx_gallery_ below for more details).
```

Comment:
```suggestion
  :ref:`_sphinx_gallery` below for more details).
```
?

Reply:
It should be
```rst
:ref:`sphinx_gallery`
```
then. I thought I would try the 'first method' (according to https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#explicit-links) but I'm happy to use `:ref:` everywhere (might be more standard, reading https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html).

Reply:
Done 69e41b51.